sessionspaces/README.md

```diff
@@ -0,0 +1,4 @@
+# SessionSpaces
+
+A service which templates out configuration (particularly namespaces)
+for each Session in ISPyB
```

Comment:
Would be good to have a little bit more detail as to what this does. I.e. what k8s resources are created, and the ISPyB tables that are referenced in doing so